core/api/v1/serializers.py

```diff
@@ -0,0 +1,9 @@
+from rest_framework import serializers
+
+from core.models import Location
+
+
+class LocationSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Location
+        fields = '__all__'
```

Comment:
Don't forget about styling: 1 blank line in every file, you can use `flake8` or `black` to lint code